src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(num) {
+    if (num === undefined) {
```

Comment:
Good use of the strict equality check '===' to ensure that 'num' is explicitly undefined. This prevents any unintended truthy or falsy evaluations.